.github/workflows/release.yml

```diff
@@ -11,6 +11,8 @@ permissions:
 jobs:
   test:
     uses: ./.github/workflows/test.yml
+    with:
+      full-matrix: true
```

Comment:
IIUC tags are generated from commits in the `main` branch, and those commits are already verified in the CI - I'd assume the matrix runs for all the merge-commits in `main`.

What's the rationale for running them again?

Reply:
Currently, the full matrix does not run on push to main. Only the partial matrix.

Currently, only the scheduled job executes the full matrix.

Reply:
> Currently, the full matrix does not run on push to main. Only the partial matrix.

What if they run by default on the main, so there is no need to run them on the releases.

There is a chance, releases might be affected by the test flakiness, and I think the release process should be as smoothy as possible if the other validations run by default.

What do you think?

Reply:
Makes sense.

Reply:
https://github.com/elastic/apm-agent-python/pull/1958/commits/6021774820d3fd25bfde33bc8e0e3aca1c04996d

Reply:
I will do the suggested comment command like in in apm-agent-nodejs in a follow-up.